### PR TITLE
Fix UI for profile page

### PR DESCRIPTION
**Pull request by madhura112**
Make elements scrollable so everything fits on the page, get rid of password settings button since it wasn't functional + wasn't part of planned work for this sprint.

### CODE REVIEW

File: Civility Optics/Views/Onboarding/profile.swift [hunk 1]
```diff
@@ -63,17 +64,66 @@ struct profile: View {
...
+                    }
+                  }.frame(maxHeight: 50)
+                  .padding()
+                  .background(
+                    RoundedRectangle(cornerRadius: 20)
+                      .foregroundColor(.init(white: 0.95)))
+                    }
+                }.frame(minHeight: 100)
+            
+//            List {
```
Comment (Spellbreeze) [inline Civility Optics/Views/Onboarding/profile.swift:114]: Should we delete this unused visual?

**Review by Spellbreeze — APPROVED** [review]

File: Civility Optics/Views/Onboarding/profile.swift [hunk 2]
```diff
@@ -63,17 +64,66 @@ struct profile: View {
...
+                                let endOfSentence = result.date_visited.firstIndex(of: "T")!
+                                let date = result.date_visited[...endOfSentence]
+                                Text(date).font(.caption)
+                                Spacer()
+                            }
+                        }
+                        HStack(spacing: 4) {
+                            Text("Rating: ")
+                                .bold()
+        //                        .multilineTextAlignment(.leading)
```
Comment (menelik-g) [inline Civility Optics/Views/Onboarding/profile.swift:85]: This line can be deleted if it serves no further purpose

File: Civility Optics/Views/Onboarding/profile.swift [hunk 3]
```diff
@@ -63,17 +64,66 @@ struct profile: View {
...
+//                HStack{
+//                    Spacer()
+//                Button{
+//                }label: {
+//                    Text("Password Settings")
+//                        .bold()
+//                }.foregroundColor(.velvet)
+//                    Spacer()
+//                }
+//            }.scrollEnabled(false)
```
Comment (menelik-g) [inline Civility Optics/Views/Onboarding/profile.swift:124]: This code can be deleted if it serves no other purpose.

File: Civility Optics/Views/Onboarding/profile.swift [hunk 4]
```diff
@@ -63,17 +64,66 @@ struct profile: View {
...
+                    }
+                    }
+                  }.frame(maxHeight: 50)
+                  .padding()
+                  .background(
+                    RoundedRectangle(cornerRadius: 20)
+                      .foregroundColor(.init(white: 0.95)))
+                    }
+                }.frame(minHeight: 100)
+
```
Comment (menelik-g) [inline Civility Optics/Views/Onboarding/profile.swift:113]: Good working functionality on the Tags buttons in the user profile page

**Review by menelik-g — APPROVED** [review]
Overall good code, just need to remove some unused lines

**Pull request merged**